src/kubernetes_cluster/proof/controller_runtime_safety.rs

```diff
@@ -129,49 +128,49 @@ pub open spec fn every_in_flight_msg_has_unique_id<K: ResourceView, E: ExternalA
     }
 }
 
-pub proof fn lemma_always_every_in_flight_msg_has_unique_id<K: ResourceView, E: ExternalAPI, R: Reconciler<K, E>>()
+pub proof fn lemma_always_every_in_flight_msg_has_unique_id()
```

Comment:
At some point, we might want to revise these lemmas to pass a `spec` to them and constrain the spec in the precondition.